server/src/main/java/org/opensearch/index/search/stats/SearchStats.java

```diff
@@ -249,6 +288,10 @@ public void writeTo(StreamOutput out) throws IOException {
             out.writeVLong(suggestCount);
             out.writeVLong(suggestTimeInMillis);
             out.writeVLong(suggestCurrent);
+
+            out.writeVLong(pitCount);
```

Comment:
```
if (out.getVersion().onOrAfter(Version.V_3_0_0)) {
    out.writeVLong(pitCount);
    out.writeVLong(pitTimeInMillis);
    out.writeVLong(pitCurrent);
}

Reply:
@reta we are targeting 2.3 OpenSearch so I need add Version.V_2_3_0 right?

Reply:
Sadly, not at this stage, the "dance" we do it like this:
 - use `V_3_0_0` for `main`
 - backport to `2.x` and replace `V_3_0_0`  use `V_2_3_0`
 - forwardport `V_2_3_0` to  `main`

The reason for that is: because your change is not in 2.3, serdes will fail now

Reply:
sure. Added `V_3_0_0` and modified the PR.